packages/p2p/src/components/orders/orders.jsx

```diff
@@ -7,11 +7,22 @@ import OrderDetails from './order-details/order-details.jsx';
 import OrderTable from './order-table/order-table.jsx';
 import './orders.scss';
 
-const Orders = ({ params }) => {
+const Orders = ({ params, navigate }) => {
     const { orders } = React.useContext(Dp2pContext);
     const [order_details, setDetails] = React.useState(null);
+    const [nav, setNav] = React.useState(params?.nav);
     const showDetails = setDetails;
-    const hideDetails = () => setDetails(null);
+    const hideDetails = () => {
+        if (nav) {
+            navigate(nav.location);
+        }
+        setDetails(null);
+    };
+    React.useEffect(() => {
+        if (params && params.nav) {
+            setNav(params.nav);
```

Comment:
Nitpick: 
```suggestion
            setNav(params?.nav ?? nav);
```
So if can be removed.

Reply:
@reza-fs 
Done!